sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -5126,7 +5137,7 @@ var setParentClass = function(child, parent) {
 
     glsBuiltinPrecisionTests.ATan2.prototype.precision = function(ctx, ret, x, y) {
         if (ctx.floatPrecision == gluShaderUtil.precision.PRECISION_HIGHP)
-            return ctx.format.ulp(ret, 6.0);
+            return ctx.format.ulp(ret, 4096.0);
```

Comment:
The native code relaxed the restriction of the precision by this code snippet: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/glshared/glsBuiltinPrecisionTests.cpp#2470